auth/tokens.go

```diff
@@ -46,11 +46,22 @@ func GetOrCreateJWTToken(ctx context.Context, user *models.Person, sessionId str
 		return token.(string), nil
 	}
 
-	token, err := jwt.NewWithClaims(jwt.SigningMethodHS256, jwt.MapClaims{
+	claims := jwt.MapClaims{
 		"role": config.Postgrest.DBRole,
 		"id":   user.ID.String(),
-	}).SignedString([]byte(config.Postgrest.JWTSecret))
+	}
+
+	if ctx.Agent() != nil {
```

Comment:
We need to update the permissions table/model to add agent and tags columns and then call GetImplicitPermissionsForUser,   We could map back to the permission table with v4/5 so that we don't need to parse the expression to get the agent/tag values.

We can't query permissions directly due to roles/groups and deny permissions